BrightFuturesTests/ResultTests.swift

```diff
@@ -151,26 +161,26 @@ class ResultTests: XCTestCase {
         XCTAssert(r.isFailure)
         XCTAssertEqual(r.error!.domain, "DivisionByZeroError")
     }
-    
-    func testRecoverNeeded() {
```

Comment:
The recover functionality is only implemented in antitypical/Result by the `??` operator. It's not possible to add recover as such, but we could just replace `recover()` with `??` here